NSS-Therapeutic-Raccoons/BangazonAPI
# Kh department

## Link to Ticket
Closes # [Link](https://github.com/NSS-Therapeutic-Raccoons/BangazonAPI/projects/6)

## Description of Proposed Changes
- Created mvc for Department

## Steps to Test:

```sh
git fetch --all
git checkout <KHDepartment>
Make sure and run latest SQL inserts
Test POST, PUT,  and GET in POSTMAN and 
filter = any amount entered 
and  ?_include=employees to bring back dept with employees
```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* Department Controller. 

## Mentions @username

@askingalot 
@jeremiah3643 
@thatmikeparrish 

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
